packages/data/src/test/privateAPIs.js

```diff
@@ -164,6 +165,43 @@ describe( 'Private data APIs', () => {
 			);
 			expect( subPrivateSelectors.getSecretDiscount() ).toEqual( 800 );
 		} );
+
+		it( 'should support registry selectors', () => {
```

Comment:
This is a passing test for a fix I included in https://github.com/WordPress/gutenberg/pull/50643.